e2e_tests/docker_image/parsec-service-test-all.Dockerfile

```diff
@@ -161,3 +161,18 @@ ENV SPIFFE_ENDPOINT_SOCKET="unix:///tmp/agent.sock"
 
 # Add safe.directory configuration to access repos freely
 RUN git config --global --add safe.directory '*'
+
+# Install latest Trusted Services libraries. The previously installed
+# libraries are old and necessary for ./generate-keys.sh which uses 
+# Parsec 1.0.0 version that is incompatible with newer libts APIs. 
+RUN rm /usr/local/lib/libts.so* /usr/local/lib/libprotobuf-nanopb.a /usr/local/lib/libmbedcrypto.a 
```

Comment:
Why are these removed?

Reply:
These are old libraries that are used for Parsec 1.0.0.

Reply:
Why are they there in the first place?

Reply:
Please have a look at the docker file from the beginning. Initially we build an old version of Parsec with an old TS libraries. After generate-keys.sh script is run we build a new version of Parsec with a new version of TS libraries.